scene/gui/spin_box.cpp

```diff
@@ -249,6 +249,9 @@ void SpinBox::_notification(int p_what) {
 			_update_text();
 		} break;
 
+		case NOTIFICATION_VISIBILITY_CHANGED:
+			drag.allowed = false;
```

Comment:
Shouldn't this only happen when the SpinBox is made invisible?

Reply:
Yeah but I'm not sure it is in fact better to check performance wise then to set it both time whatever the case if it is invisible it should already be set to false. I think checking if it is in tree will be more costly then just setting the value. But I might be thinking badly about it feel free to correct me if it is the case

Reply:
`is_visible_in_tree()` is a simple bool check, the value is cached.

Reply:
Always thought reading and writing bool value was the same time but I might be wrong moreover here we will have one check when hiding before setting which means for it to be worth it the function needs to always be twice faster the setting the value which I felt it was most likely the case

Reply:
Well, the code I suggested also prevents calling `_release_mouse()`.
Although after reading it again, it doesn't really matter if the check exists or not.